src/utils.rs

```diff
@@ -516,3 +516,64 @@ pub fn parse_from_cli(key: &str) -> Option<String> {
         .and_then(|idx| args.get(idx + 1))
         .map(String::to_string)
 }
+
+/// Remove path: It can be directory or file
+pub async fn remove(path: &camino::Utf8PathBuf) -> std::io::Result<()> {
+    if path.is_file() {
+        tokio::fs::remove_file(path).await?;
+    } else if path.is_dir() {
+        tokio::fs::remove_dir_all(path).await?
+    } else if path.is_symlink() {
+        // TODO:
+        // It can be a directory or a file
+    }
+    Ok(())
+}
+
+/// Remove from provided `root` except given list
+pub async fn remove_except(root: &camino::Utf8PathBuf, except: &[&str]) -> fpm::Result<()> {
```

Comment:
Writing `&camino::Utf8PathBuf` is like writing `String`. The equivalent of `&str` is `&camino::Utf8Path`.